base/util_testing.go

```diff
@@ -173,13 +172,28 @@ func TestUseXattrs() bool {
 	useXattrs := os.Getenv(TestEnvSyncGatewayUseXattrs)
 	switch {
 	case strings.ToLower(useXattrs) == strings.ToLower(TestEnvSyncGatewayTrue):
-		Infof(KeyAll, "Using xattrs: strings.ToLower(useXattrs) == strings.ToLower(TestEnvSyncGatewayTrue).  |%v| == |%v|", strings.ToLower(useXattrs), strings.ToLower(TestEnvSyncGatewayTrue))
 		return true
 	default:
-		Infof(KeyAll, "NOT Using xattrs: strings.ToLower(useXattrs) != strings.ToLower(TestEnvSyncGatewayTrue).  |%v| != |%v|", strings.ToLower(useXattrs), strings.ToLower(TestEnvSyncGatewayTrue))
+		// Otherwise fallback to hardcoded default
+		return DefaultUseXattrs
```

Comment:
Good idea to clean this up, but it looks like we can go a bit further and convert to a simple if statement.

Reply:
Fixed